pure-pso/values.yaml

```diff
@@ -95,14 +98,19 @@ images:
       pullPolicy: Always
     cockroachOperator:
       name: purestorage/cockroach-operator
-      tag: v1.0.3
+      tag: v1.0.4-smartagent-1
```

Comment:
change back

Reply:
Yes. I'll change back later. However, I leave it for now for the testing purpose.